core/sql/optimizer/NATable.h

```diff
@@ -576,7 +576,17 @@ class NATable : public NABasicObject
   }
 
   NABoolean isSQLMXAlignedTable() const
-  {  return (flags_ & SQLMX_ALIGNED_ROW_FORMAT) != 0; }
+  {
```

Comment:
Now that we have file formats per index, we should move this method from NATable to NAFileSet, for example as NAFileSet::isSQLMXAlignedFormat(). Instead of calling this method on an NATable *t, the caller should then call t->getClusteringIndex()->isSQLMXAlignedFormat(). I would also have some comments on the implementation below, but they are irrelevant if you replace this method.